include/dftd4.h

```diff
@@ -127,6 +132,14 @@ dftd4_custom_d4_model(dftd4_error /* error */,
                       double /* gc */,
                       double /* wf */) DFTD4_API_SUFFIX__V_3_1;
 
+/// Create new D4 dispersion model
+DFTD4_API_ENTRY dftd4_model DFTD4_API_CALL
+dftd4_custom_d4s_model(dftd4_error /* error */,
+                      dftd4_structure /* mol */,
+                      double /* ga */,
+                      double /* gc */) DFTD4_API_SUFFIX__V_3_1;
```

Comment:
Same here the API suffix should indicate the version the API was added or changed